updates/P6.txt

```diff
@@ -0,0 +1,135 @@
+                           Protocol update
+
+Protocol Version: 6
+Builds on Protocol Version: 5
+
+                               Abstract
+
+This document describes the changes in Concordium protocol version 6
+compared to protocol version 5.
+
+The protocol change has the following main effects
+- it changes the consensus protocol from an existing two-layer design
+  to a new Concordium BFT consensus
+- fixes the behaviour of state rollbacks in specific scenarios with
+  version 1 contracts.
+- extends the list of supported Wasm instructions for version 1
+  contracts, and fixes an incompatibility with the Wasm standard
+- adds better support for sponsored transactions
+
+                              Background
+
+Protocol version 6 changes protocol version 5 in two main areas.
+The first area is a relatively modest extension of smart contract
+functionality, with the main effect being better support for
+sponsored transactions.
+The major update in protocol 6 however, is the new consensus protocol,
+Concordium BFT.
+
+Up to and including protocol 5, Concordium used a two-layer consensus
+design, with a proof of stake Nakamoto style consensus at the base
+layer with finalization on top which marked certain blocks as finalized.
+The idea being that the base Nakamoto layer works as long as more than
+1/2 of stake adheres to the protocol over long periods of time, but
+the finalization layer ensures speedy, explicit, finalization, in good
+conditions when there is more than 2/3 liveness and adherence to protocol.
+
+While this two-layer design did serve Concordium well during the initial
+phase, and is in theory robust, the two layer design is rather complex,
+and makes it challenging to optimize the consensus for both throughput and
+confirmation latency while maintaining security.
+
+In protocol 6 a new consensus protocol Concordium Byzantine Fault Tolerance
+(BFT) will be used instead. The protocol offers high transaction throughput
+and lower confirmation time because a block can be produced as soon as the
+previous block has been signed. The overall architecture remains the same.
```

Comment:
"certified" would probably be more precise than "signed" since one signature is not enough.